src/components/Form/Form.js

```diff
@@ -15,15 +17,14 @@ export default function Form() {
 	} = useForm();
 
 	const onSubmit = book => {
-		addNewBook(book);
+		const newBook = {
+			...book,
+			bookmarkStatus: bookmarkStatus,
+		};
+		addNewBook(newBook);
```

Comment:
you could just fill it like this, or is there any reason to use an extra namespace for that? :

```
		addNewBook({
			...book,
			bookmarkStatus: bookmarkStatus,
		});
```

then optimize a little:

```
		addNewBook({
			...book,
			bookmarkStatus,
		});
```